website/docs/r/instance.html.markdown

```diff
@@ -130,8 +130,8 @@ The following attributes are exported:
 
 ## Import
 
-Servers can be imported using the server `ID`, e.g.
+Instances can be imported using the server `ID`, e.g.
```

Comment:
Thanks for submitting the PR to fix this.

Can we also update `using the server` to `using the instance` while we are here?